includes/Jobs/RequestSSLCFAddJob.php

```diff
@@ -0,0 +1,412 @@
+<?php
+
+namespace Miraheze\RequestSSL\Jobs;
+
+use Exception;
+use Job;
+use MediaWiki\Config\Config;
+use MediaWiki\Context\RequestContext;
+use MediaWiki\Http\HttpRequestFactory;
+use MediaWiki\Logger\LoggerFactory;
+use MediaWiki\MainConfigNames;
+use MediaWiki\MediaWikiServices;
+use MediaWiki\User\User;
+use MessageLocalizer;
+use Miraheze\RequestSSL\RequestSSLManager;
+use Psr\Log\LoggerInterface;
+
+class RequestSSLCFAddJob extends Job {
+
+	public const JOB_NAME = 'RequestSSLCFAddJob';
+
+	private readonly Config $config;
+	private readonly LoggerInterface $logger;
+	private readonly MessageLocalizer $messageLocalizer;
+
+	private readonly string $apiKey;
+	private readonly string $baseApiUrl;
+	private readonly int $id;
+	private readonly User $systemUser;
+	private readonly string $zoneId;
+
+	public function __construct(
+		array $params,
+		private readonly HttpRequestFactory $httpRequestFactory,
+		private RequestSSLManager $requestSSLManager
```

Comment:
just calling it $requestManager makes since since this is RequestSSL, also makes it easier for when we end up renaming the extension. Also make readonly.